src/issues/mod.rs

```diff
@@ -103,15 +102,14 @@ impl<C: Clone + Connect> IssueLabels<C> {
             github: github,
             owner: owner.into(),
             repo: repo.into(),
-            which: which,
             number: number,
         }
     }
 
     fn path(&self, more: &str) -> String {
         format!(
             "/repos/{}/{}/{}/{}/labels{}",
-            self.owner, self.repo, self.which, self.number, more
+            self.owner, self.repo, "issues", self.number, more
```

Comment:
I suppose this could probably be reverted to the way it was before #121, with the issues inside of the first string.

Reply:
nice catch! Ill fix that merge and publish this weekend